rules_gapic/disco_to_proto.bzl

```diff
@@ -44,6 +45,7 @@ proto_from_disco = rule(
         "service_ignorelist": attr.string_list(allow_empty = True, default = []),
         "message_ignorelist": attr.string_list(allow_empty = True, default = []),
         "relative_link_prefix": attr.string(mandatory = False, default = ""),
+        "enums_as_strings": attr.bool(mandatory = False, default = False),
```

Comment:
If we'e GAing with this, it may make sense to make this default to True.

But at any rate, we should be explicit with this flag in any invocations, so maybe it's not a big deal.

Reply:
I will strongly prefer keeping enums the default, because this is the desired behavior and the reason why we ship without it is purely due to the current limitations in enum json representation which we hope to fix in the mid-term future.